README.md

```diff
@@ -30,20 +65,246 @@ public function registerBundles()
+                    $data,
+                    $remoteId,
+                );
+            }
+
+
+            return $struct;
+        });
+        $builder->addWriter($this->contentWriter);
+    }
+}
+```
+
+# Admin UI
+
+## Access to the eZ Dataflow UI
+
+You can access to the eZ Dataflow administation UI from your eZ Platform admin back-office.
+
+![Admin menu](src/Resources/doc/ez_dataflow_admin_menu.jpg)
+
+1. Click to "Admin"
+1. Click to "eZ Dataflow"
+
+## Add a new schedule
+
+Go to the eZ Dataflow admin, and click on the "+" orange button. 
+
+Il the new popin, complete the fields:
+
+![Add a new schedule](src/Resources/doc/add_new_schedule.jpg)
+
+1. Type the Dataflow name
+1. Select the Dataflow type. The list is automaticaly generated from the list of Symfony service with the tags `coderhapsodie.dataflow.type`. If your dataflow type is not present, [check the configuration](https://github.com/code-rhapsodie/dataflow-bundle#check-if-your-dataflowtype-is-ready)
+1. Type here the Dataflow options. Format: one option per line and option name and value with `:` separator.
+1. Type here the frequency. The value must be compatible with the the PHP [strtotime](https://www.php.net/manual/en/function.strtotime.php) function.
+1. Choice the date and time of the first job.
+1. Check if you want run this Dataflow.
+
+Finally, click on the "Create" button.
+
+## Read the history
+
+When you click on the "History" tab in the eZ Dataflow admin UI, you view the job history for all Dataflow configured and executed.
+
+![History list](src/Resources/doc/history_list.jpg)
+
+1. The history list
+1. This column shows the number of objects that have been processed.
+1. Click on the question mark for display the job details.
+
+Details of one scheduled job:
+
+![Job execution details](src/Resources/doc/job_successful.jpg)
+
+## One-shot job
+
+If you don't want run a Dataflow periodically, you can add a single execution at the time and date what you want.
+
+Go to the eZ Dataflow admin UI and click on the "Oneshot" tabs.
+
+![Oneshot list](src/Resources/doc/oneshot_list.jpg)
+
+1. This button allows you to define the one-shot job (see below).
+1. This column shows the number of objects that have been processed.
+1. Click on the question mark to display the job details.
+
+Details of one-shot job:
+
+![onshot details](src/Resources/doc/job_fail.jpg)
+
+Here the job has fail.
+
+## Add a one-shot job
+
+Go to the eZ Dataflow admin UI and click on the "Oneshot" tabs. Click to the "+" orange button to open the adding popin.
+
+![The add one-shot popin](src/Resources/doc/one_shot_job.jpg)
+
+1. Type the Dataflow name
+1. Select the Dataflow type. The list is automaticaly generated from the list of Symfony service with the tags `coderhapsodie.dataflow.type`. If your dataflow type is not present, [check the configuration](https://github.com/code-rhapsodie/dataflow-bundle#check-if-your-dataflowtype-is-ready)
```

Comment:
```suggestion
1. Select the Dataflow type. The list is automatically generated from the list of Symfony service with the tags `coderhapsodie.dataflow.type`. If your dataflow type is not present, [check the configuration](https://github.com/code-rhapsodie/dataflow-bundle#check-if-your-dataflowtype-is-ready)
```

Reply:
Aren't these explanations already above ?

Reply:
Yes, the forms to adding a new scheduled dataflow or one-shot dataflow have the same fields.